### PR TITLE
IMXRT1050-EVK: Hello world application fails to be downloaded

### PR DESCRIPTION
**Pull request by btheosam**
imxrt1050-evk_download.sh misses logic to convert kB to bytes

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [b0e0b381bfd97ce60b19397b1d2e55d59bc86cc5] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [b0e0b381bfd97ce60b19397b1d2e55d59bc86cc5] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [7a06fca539a81bfe746ebf761ec53754b781cee2] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [7a06fca539a81bfe746ebf761ec53754b781cee2] - Code Rule Check OK.

File: build/configs/imxrt1050-evk/imxrt1050-evk_download.sh [hunk 1]
```diff
@@ -57,8 +57,8 @@ SUDO=sudo
 
 # Following to read partition information
 PARTITION_KCONFIG=${OS_PATH}/board/common/Kconfig
-source ${OS_PATH}/.config
```
Comment (btheosam) [inline build/configs/imxrt1050-evk/imxrt1050-evk_download.sh]: Duplication of sourcing config, so can be removed.

**Pull request merged**